Add `BN Mentorship: Cycle 7 Sign-ups Open` news post (#14880)
ppy/osu-wiki: news/2026/2026-06-23-bn-mentorship-cycle-7.md

---
layout: post
title: "BN Mentorship: Cycle 7 Sign-ups Open"
date: 2026-06-23 21:00:00 +0000
---

There's no perfect time to become a Beatmap Nominator other than now, and BN Mentorship returns to help you on that journey!

![](/wiki/shared/news/banners/bn-mentorship/banner.jpg)

> For those who haven't seen one of these before, here's the usual spiel: this is the 7th cycle of the BN Mentorship Program, an ongoing system led by the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) to try and teach any up-and-coming modders who wish to join the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) the skills and insight they may be lacking to do so.

Reviewer: ```suggestion
For those who haven't seen one of these before, here's the usual spiel: this is the 7th cycle of the BN Mentorship Program, an ongoing system led by the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) to teach up-and-coming modders the skills needed to become [Beatmap Nominators](/wiki/People/Beatmap_Nominators).
```
i struggled to read the original version

> For this cycle, all game modes are welcome to sign up. Check out a list of mentors below:

Reviewer: ```suggestion
For this cycle, all game modes are welcome to sign up.
```
the next line already says "Below you'll find a list of mentors"

### ![osu!catch](/wiki/shared/mode/catch.png "osu!catch") osu!catch mentors

| Mentor | Timezone |
| :-- | :-- |
| ::{ flag=HK }:: [BlackBN](https://osu.ppy.sh/users/6291741) / ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776) | UTC+8 / UTC+2 |

### ![osu!mania](/wiki/shared/mode/mania.png "osu!mania") osu!mania mentors

| Mentor | Timezone |
| :-- | :-- |
| ::{ flag=VN }:: [Akasha-](https://osu.ppy.sh/users/2596306) / ::{ flag=RU }:: [Kurisu Makise](https://osu.ppy.sh/users/2790640) | UTC+7 / UTC+3 |
| ::{ flag=VN }:: [[TCD] Dzar03](https://osu.ppy.sh/users/16712231) / ::{ flag=VN }:: [[TCD] Ena-suki](https://osu.ppy.sh/users/29645291) | UTC+7 |
| ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773) | UTC-4 |
| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) | UTC+10 |
| ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534) | UTC+8 |
| ::{ flag=NL }:: [Muse Dash](https://osu.ppy.sh/users/13695676) | UTC+2 |
| ::{ flag=ID }:: [Pyra-san](https://osu.ppy.sh/users/31126430) | UTC+7 |

---

[![Click here to apply for BN Mentorship!](/wiki/shared/news/banners/bn-mentorship/apply.png)](https://forms.gle/Nx53WnAkMbpEaWor5)

> Signups are slated to be open from **23 June** until **28 June**, while the cycle itself will start on **29 June** and will end on **30 August**.

Reviewer: ```suggestion
Signups are slated to be open from **June 23rd** until **June 28th**, while the cycle itself will start on **June 29th** and will end on **August 30th**.
```
i think writing dates in a conversational way reads better than the formal way. looks like previous mentorship articles used dates in this way too
